tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -194,13 +248,13 @@ static void le_security_changed(struct bt_conn *conn, bt_security_t level,
 		 * This means bond is lost and we restart pairing to re-bond
 		 */
 		if (bt_conn_get_info(conn, &info) == 0 &&
-		    info.role == BT_CONN_ROLE_CENTRAL) {
+			info.role == BT_CONN_ROLE_CENTRAL) {
```

Comment:
This also seems unrelated and wrong. Split lines should be aligned right after the opening parenthesis of the previous line, using 0-N tabs + 0-7 spaces. Looks to me like the line was already correct before this change.